solitude/settings/base.py

```diff
@@ -19,14 +19,6 @@
 #
 ALLOWED_HOSTS = []
 
-if os.environ.get('MEMCACHE_URL'):
-    CACHES = {
-        'default': {
-            'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-            'LOCATION': os.getenv('MEMCACHE_URL'),
-        }
-    }
-
```

Comment:
Oh, we don't use memcache in prod at all? If that's the case then disregard my comment. What I meant was that if prod is using memcache then it must be doing so via the environment variable since it's not explicitly doing so in the settings.